frescobaldi_app/lilypondinfo.py

```diff
@@ -183,6 +183,19 @@ def abscommand(self):
             path = glob.glob(os.path.join(
                 os.environ.get('ProgramFiles', 'C:\\Program Files'),
                 'LilyPond*', 'usr', 'bin'))
+            if not path:
```

Comment:
I reviewed all the changes so far, found this one instance of a new feature not directly related to the PyQt upgrade. Looks OK though.

Reply:
You caught me. Mixed-architecture setups like mine (64-bit Python and Frescobaldi, but 32-bit LilyPond) are a relatively common Weird Windows Thing™. I'm guessing this hasn't come up before because historically both LilyPond and Frescobaldi have been built as 32-bit only on Windows.